packages/cpp-debug/src/browser/wrapper-widgets/memory-layout-widget.tsx

```diff
@@ -87,6 +87,7 @@ export class MemoryLayoutWidget extends Panel implements Disposable, Application
 
     dispose(): void {
         this.toDispose.dispose();
+        super.dispose();
```

Comment:
I think this line was just forgotten. With the original version it was not possible to dispose of this widget.